src/libnetdata/locks/locks.h

```diff
@@ -9,16 +9,27 @@
 typedef pthread_mutex_t netdata_mutex_t;
 #define NETDATA_MUTEX_INITIALIZER PTHREAD_MUTEX_INITIALIZER
 
+#if 0
 typedef struct netdata_spinlock {
     bool locked;
 #ifdef NETDATA_INTERNAL_CHECKS
     pid_t locker_pid;
     size_t spins;
 #endif
 } SPINLOCK;
+#else
+typedef struct netdata_spinlock {
+    netdata_mutex_t inner;
```

Comment:
When we cross compile our code on Linux, `netdata_mutex_t` is one of the issues raised. Is there the possibility we already start using Windows native functions and data type? Or to use `libuv`?